README.md

```diff
@@ -68,6 +68,24 @@ make kbs-coco-as-grpc
 The AS `gRPC` address can be specified in the [KBS config file](https://github.com/confidential-containers/kbs/blob/main/src/api/src/config.rs),
 and by default KBS will try to reach a locally running AS at `127.0.0.1:50004`.
 
+### `Amber` Attestation Service
+
+KBS supports Amber as the Attestation Service with the `amber-as` Cargo feature.
```

Comment:
@Lu-Biao Is there any public API documentation that we could have that document point to?